cmd/worker/main.go

```diff
@@ -20,12 +37,35 @@ func new_job(w http.ResponseWriter, req *http.Request) {
 	// Put the bytes from the request into a file
 	buf := new(bytes.Buffer)
 	buf.ReadFrom(req.Body)
-	file := buf.String()
+	jJson := buf.String()
```

Comment:
On first glace, this name is senseless. I think we should change it to jobJson.

Reply:
I changed some naming @showalter